yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/filters/RequiredDocumentBuilder.java

```diff
@@ -0,0 +1,9 @@
+package com.yoti.api.client.docs.session.create.filters;
+
+public interface RequiredDocumentBuilder<Builder extends RequiredDocumentBuilder<Builder, Filter>, Filter extends RequiredDocumentFilter> {
+
+    Builder withFilter(Filter filter);
```

Comment:
Should this method be removed from this interface now that it's ID document specific?

Reply:
yes.  on it.